### PR TITLE
Render version 1.6

### PR DESCRIPTION
**Pull request by anmazzotti**

### CODE REVIEW

File: versioned_docs/version-1.5/release-notes.md [hunk 1]
```diff
@@ -17,10 +17,10 @@ Here's the different components, their latest version and a link to the respecti
 
 | Name                                                                 | Version | Release Notes                                                              |
 |----------------------------------------------------------------------|---------|----------------------------------------------------------------------------|
-| [Elemental Operator](https://github.com/rancher/elemental-operator/) | v1.5.4  | [Link](https://github.com/rancher/elemental-operator/releases/tag/v1.5.4)  |
-| [Elemental Toolkit](https://github.com/rancher/elemental-toolkit/)   | v1.1.5  | [Link](https://github.com/rancher/elemental-toolkit/releases/tag/v1.1.5)   |
+| [Elemental Operator](https://github.com/rancher/elemental-operator/) | v1.6.3  | [Link](https://github.com/rancher/elemental-operator/releases/tag/v1.6.3)  |
```
Comment (davidcassany) [inline versioned_docs/version-1.5/release-notes.md:20]: These release notes for v1.5 should remain unchanged.

File: versioned_docs/version-1.6/release-notes.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+Each of these components have an independent development lifecycle with its own versioning. Once a new version is ready, meaning it fully integrates with the others components of the Elemental project stack, a release is made.
+
+Here's the different components, their latest version and a link to the respective release notes on GitHub:
+
+| Name                                                                 | Version | Release Notes                                                              |
+|----------------------------------------------------------------------|---------|----------------------------------------------------------------------------|
+| [Elemental Operator](https://github.com/rancher/elemental-operator/) | v1.6.3  | [Link](https://github.com/rancher/elemental-operator/releases/tag/v1.6.3)  |
+| [Elemental Toolkit](https://github.com/rancher/elemental-toolkit/)   | v2.1.1  | [Link](https://github.com/rancher/elemental-toolkit/releases/tag/v2.1.1)   |
+| [Elemental Linux](https://github.com/rancher/elemental)              | v2.0.4  | [Link](https://github.com/rancher/elemental/releases/tag/v2.0.4)           |
```
Comment (davidcassany) [inline versioned_docs/version-1.6/release-notes.md:22]: These should be on version v2.1.1, current OS versions releases are at v2.1.1

**Review by davidcassany — CHANGES_REQUESTED** [review]
Release notes are not correct for v1.5 (I guess they were changed by mistake) and the v1.6 ones require an adjustment of the OS version.

Apart from that all looks good to me.

File: versioned_docs/version-1.5/release-notes.md (hunk 1 above)
Comment (anmazzotti) [reply versioned_docs/version-1.5/release-notes.md:20]: Good catch, I meant to update the "next" ones, not these. :D

File: versioned_docs/version-1.6/release-notes.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+Each of these components have an independent development lifecycle with its own versioning. Once a new version is ready, meaning it fully integrates with the others components of the Elemental project stack, a release is made.
+
+Here's the different components, their latest version and a link to the respective release notes on GitHub:
+
+| Name                                                                 | Version | Release Notes                                                                |
+|----------------------------------------------------------------------|---------|------------------------------------------------------------------------------|
+| [Elemental Operator](https://github.com/rancher/elemental-operator/) | v1.6.3  | [Link](https://github.com/rancher/elemental-operator/releases/tag/v1.6.3)    |
+| [Elemental Toolkit](https://github.com/rancher/elemental-toolkit/)   | v2.1.1  | [Link](https://github.com/rancher/elemental-toolkit/releases/tag/v2.1.1)     |
+| [Elemental Linux](https://github.com/rancher/elemental)              | v2.0.4  | [Link](https://github.com/rancher/elemental/releases/tag/v2.1.1)             |
```
Comment (davidcassany) [inline versioned_docs/version-1.6/release-notes.md:22]: This elemental repo (OS image version) should be on v2.1.1

Comment (anmazzotti) [reply versioned_docs/version-1.6/release-notes.md]: > This elemental repo (OS image version) should be on v2.1.1

Three time's a charm. :D
Should be fixed now, thank you.

File: versioned_docs/version-1.6/release-notes.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+Here's the different components, their latest version and a link to the respective release notes on GitHub:
+
+| Name                                                                 | Version | Release Notes                                                                |
+|----------------------------------------------------------------------|---------|------------------------------------------------------------------------------|
+| [Elemental Operator](https://github.com/rancher/elemental-operator/) | v1.6.3  | [Link](https://github.com/rancher/elemental-operator/releases/tag/v1.6.3)    |
+| [Elemental Toolkit](https://github.com/rancher/elemental-toolkit/)   | v2.1.1  | [Link](https://github.com/rancher/elemental-toolkit/releases/tag/v2.1.1)     |
+| [Elemental Linux](https://github.com/rancher/elemental)              | v2.1.1  | [Link](https://github.com/rancher/elemental/releases/tag/v2.1.1)             |
+| [Elemental UI](https://github.com/rancher/elemental-ui)              | v1.3.1  | [Link](https://github.com/rancher/elemental-ui/releases/tag/elemental-1.3.1) |
```
Comment (fgiudici) [inline versioned_docs/version-1.6/release-notes.md:23]: This is not existent yet, but makes sense as 1.3.1 should be out soon I guess 👍🏼

**Review by fgiudici — APPROVED** [review]
LGTM

**Review by davidcassany — APPROVED** [review]
LGTM Thanks!

**Pull request merged**